src/platform/plugins/private/kibana_usage_collection/server/collectors/saved_objects_counts/get_saved_object_counts/get_saved_object_counts.ts

```diff
@@ -80,16 +80,18 @@ export async function getSavedObjectsCounts(
   const nonExpectedTypes: string[] = [];
 
   const perType = buckets.map((perTypeEntry) => {
-    if (perTypeEntry.key !== MISSING_TYPE_KEY && !soTypes.includes(perTypeEntry.key)) {
+    const key = perTypeEntry.key as string;
+    if (key !== MISSING_TYPE_KEY && !soTypes.includes(key)) {
       // If the breakdown includes any SO types that are not expected, highlight them in the nonExpectedTypes list.
-      nonExpectedTypes.push(perTypeEntry.key);
+      nonExpectedTypes.push(key);
     }
 
     return { key: perTypeEntry.key, doc_count: perTypeEntry.doc_count };
   });
 
   return {
     total: body.total,
+    // @ts-expect-error `FieldValue` types now claim that bucket keys can be `null`
```

Comment:
Oof, can this be true?

Reply:
I checked and it cannot actually be true. null values are not indexed by Elasticsearch, and can't be bucket keys. Even the `missing` parameter of aggregations forbids null values. For booleans, the key is mapped to an integer (and `key_as_string` is set). I did not know this, sorry.

So, for bucket keys, we could use `export type FieldValue = long | double | string`. I'll see how this can be fixed in the Elasticsearch specification.